google-cloud-bigtable/lib/google/cloud/bigtable/read_operations.rb

```diff
@@ -320,6 +322,11 @@ def build_row_set row_keys, row_ranges
             row_set[:row_ranges] = row_ranges.map(&:to_grpc)
           end
 
+          # Set the row range to full table scan if the row set is empty
+          if row_set.empty?
+            return Google::Cloud::Bigtable::V2::RowSet.new row_ranges: []
```

Comment:
Does this actually work? In other clients we needed to insert an infinite range and then would check if row_ranges is empty. I dont quite understand what having an empty row range list gives you here